### PR TITLE
Rachit Tandon

### PR DESCRIPTION
**Pull request by webdevrachit**
Screenshot (14)

### CODE REVIEW

**Review by w3bdesign — CHANGES_REQUESTED** [review]
Needs a } at the end

File: data.json [hunk 1]
```diff
@@ -514,5 +514,10 @@
     "githubUsername": "diyajaiswal11",
     "imageUrl": "https://user-images.githubusercontent.com/48059117/106366095-a539be80-635f-11eb-95b0-c2d68c787410.png",
     "issueNumber": 285
-  }
+  },
+  {
+    "name": "Rachit Tandon",
+    "githubUsername": "webdevrachit",
+    "imageUrl": "https://user-images.githubusercontent.com/72686780/106394782-64f04400-6424-11eb-969c-b243f71209ae.png",
+    "issueNumber": 295
```
Comment (AllanRegush) [inline data.json:522]: Looks like there is a missing closing bracket `}`

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1: Just a minor adjustment. :nerd_face:

**Comment by AllanRegush** [comment]
Merged #290, Thank you for the PR :+1:  - this is a duplicate PR, Closing :nerd_face:

**Pull request closed without merging**